Allow different compile arguments to be used for programs and plugins
Index: kismet-2011-03-R2/Makefile.inc.in
===================================================================
--- kismet-2011-03-R2.orig/Makefile.inc.in	2009-11-06 03:43:30.000000000 +0100
+++ kismet-2011-03-R2/Makefile.inc.in	2012-12-03 04:12:04.285353779 +0100
@@ -22,7 +22,7 @@
 CLIBS   	= @CLIBS@
 CXXLIBS		= @CXXLIBS@
 KSLIBS		= @KSLIBS@
-CFLAGS		= @CPPFLAGS@
+CFLAGS		= @CPPFLAGS@ @CFLAGS@
 CLIENTLIBS 	= @CLIENTCLIBS@
 CAPLIBS		= @caplibs@
 CLIENTFLAGS = @CLIENTCFLAGS@
